### DISCUSSION TITLE
LDAP for non-docker installations

### DISCUSSION THREAD

**Discussion by kimizumi1** [inventree/InvenTree#6001] [Q&A]
Hi!
I noticed there is LDAP support in the newest docker version of InvenTree.
I was wondering if there is going to be an official LDAP support for non-docker installations?
Really looking forward to this feature

**Comment by wolflu05** [comment]
Hey @kimizumi1 ,

currently LDAP is only supported in docker because I only knew how to setup the required native dependencies and python packages in docker. If you have some ideas how to implement that in the installer version feel free to propose a PR. The tricky thing is that the python packages already require the native ldap packages on install, so we cannot just say we ship the python packages always and if ldap is required tell the user to install the native dependencies via apt manually. And if we always install the native apt packages, the install size is very big and this clutters the servers of users who don't need ldap support.
